.eslintrc.yml

```diff
@@ -73,5 +73,6 @@ rules:
 
   # React rules
   react/react-in-jsx-scope: off
+  react/prop-types: off
```

Comment:
Not sure about this, do we have a concrete example where it was an issue?

Reply:
```tsx
interface Props extends PropsWithChildren {
  action: () => void
  disabled?: boolean
}

export const Button: React.FunctionComponent<Props> = ({
  action, // 'action' is missing in props validation eslint(react/prop-types)
  disabled, // 'disabled' is missing in props validation eslint(react/prop-types)
  children, // 'children' is missing in props validation eslint(react/prop-types)
}) => {
  return (
    <button onClick={action} disabled={disabled}>
      {children}
    </button>
  )
}
```

It requires: 

```tsx
export const Button: React.FunctionComponent<Props> = ({ action, disabled, children }: Props) => {
  return (
    ...
  )
}
```

Which is inferred by the generic anyway.

Also, not passing the generic and still typing the argument results in an error : 
```txt
Types of parameters '__0' and 'props' are incompatible.
    Property 'action' is missing in type '{}' but required in type 'Props'.
```

Reply:
Just do this:

```tsx

interface Props {
  action: () => void
  disabled?: boolean
  children: React.ReactNode
}

export const Button: React.FC<Props> = ({ action, disabled, children }) => {
  return (
    <button onClick={action} disabled={disabled}>
      {children}
    </button>
  )
}
```

It's better to be explicit about whether the components accepts/requires children anyway

Reply:
This changes nothing. The problem is not about `children`. Just the eslint rule does not take into account the generic to `React.FonctionComponent`/`React.FC`, generic which was given in my example.

Reply:
What's the problem? Seems to work just fine:
<img width="612" alt="Capture d’écran 2022-07-30 à 14 28 24" src="https://user-images.githubusercontent.com/60700958/181914257-a75e3407-b25f-44af-8f83-b05db314078d.png">

Reply:
Doesn't work on my machine 🤷‍♂️
With the config in this pr + `react/prop-types: off` commented out.

Reply:
I see no reason why it would not work. Maybe some conflicts with your machine environment (e.g. a local `eslint` used instead of the package one)?

I suggest we set up the CI in this PR too and give it the final word 🤷

Reply:
I have no eslint on my machine other that project wide.

Sure I'll do that.

Reply:
`rm -rf ./node_modules && npm i`?

Reply:
Update : as the scope of this pr does not produce error (no such component in this version yet) I remove the rule disabling.

We will advise later.